packages/components/src/index.html

```diff
@@ -49,15 +49,9 @@
         //clientKey: "YzQzMGM3OGYtYzQ2NC00NTdjLThkM2QtNWIwYzI4OTdmYmE5",
         //userId: "smokey12345",
 
-        components: {
-          inbox: {
-            title: "foo",
-            openLinksInNewTab: false,
-            labels: {
-              closeInbox: "closme",
-              markAllAsRead: "markallread",
-            },
-          },
+        userId: "70f6a4f4-2907-4518-b8f3-b9cfab224764",
+        onRouteChange: (route) => {
+          console.log("route", route);
```

Comment:
@rileylnapier Did you mean to have a console log here?

Reply:
This is a dev file so it's fine.